extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -55,10 +66,14 @@
 .i-amphtml-story-interactive-mid-selection .i-amphtml-story-interactive-slider-bubble {
   --scale-var: calc(var(--fraction) + 1) !important;
   /* Move the bubble up by sqrt(2)/2=70.7% to account for the pointy corner */
-  transform: translate(-50%, calc(-160% - var(--fraction) * 70.7%))
+  transform: translate(var(--translate-x-center), calc(-160% - var(--fraction) * 70.7%))
     scale(var(--scale-var)) !important;
 }
 
+/*[dir='rtl'].i-amphtml-story-interactive-mid-selection .i-amphtml-story-interactive-slider-bubble {
+  --translate-x-center: 50%;
+}*/
```

Comment:
If the other review comments make it work, we can remove these commented rules and merge it